### PR TITLE
fix light flashes at instanced model for v3.5.1

### PR DESCRIPTION
**Pull request by troublemaker52025**
Re: #

### Changelog

* 

-------

### Continuous Integration

This pull request:

* [x] needs automatic test cases check // Manual trigger with `@cocos-robot run test cases` afterward

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by troublemaker52025** [comment]
* https://github.com/cocos/3d-tasks/issues/12356

**Comment by troublemaker52025** [comment]
@cocos-robot run test cases

File: native/cocos/renderer/pipeline/RenderAdditiveLightQueue.cpp [hunk 1]
```diff
@@ -400,14 +421,19 @@ bool RenderAdditiveLightQueue::getLightPassIndex(const scene::Model *model, vect
 
 void RenderAdditiveLightQueue::lightCulling(const scene::Model *model) {
     bool isCulled = false;
+    const auto isNeedCulling = !isInstancedOrBatched(model);
```
Comment (troublemaker52025) [inline native/cocos/renderer/pipeline/RenderAdditiveLightQueue.cpp:424]: * 暂时屏蔽 instanced model 还有 batched model 的裁剪。因为这两块内容的裁剪，以前一直都没有人做。留以后优化
* https://github.com/cocos/3d-tasks/issues/12363#issuecomment-1134443602

**Review by jk20012001 — APPROVED** [review]

**Review by pandamicro — APPROVED** [review]

File: cocos/core/pipeline/render-additive-light-queue.ts [hunk 2]
```diff
@@ -89,6 +89,25 @@ function getLightPassIndices (subModels: SubModel[], lightPassIndices: number[])
...
+        for (let p = 0; p < passes.length; ++p) {
+            const pass = passes[p];
+            const batchingScheme = pass.batchingScheme;
+            if (batchingScheme === BatchingSchemes.INSTANCING) {
+                return true;
+            } else if (batchingScheme === BatchingSchemes.VB_MERGING) {
+                return true;
+            } else {
+                return false;
+            }
```
Comment (minggo) [inline cocos/core/pipeline/render-additive-light-queue.ts:105]: Do not quite understand the logic. It seems it will return at first loop.

Comment (troublemaker52025) [reply cocos/core/pipeline/render-additive-light-queue.ts:105]: * https://github.com/cocos/cocos-engine/pull/11205

**Pull request merged**